test/xpu_api/numerics/complex.number/ccmplx/ccomplex.pass.cpp

```diff
@@ -6,7 +6,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "support/test_support.h"
```

Comment:
Could you please short explanations why that header is not needed any more.

Reply:
Sure.
This header simple absent in our code base. And it's name used here by error.